CHANGELOG.md

```diff
@@ -125,6 +125,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Compatibility with OpenSearch 2.1.0 - 2.4.1 ([#257](https://github.com/opensearch-project/opensearch-py/pull/257))
 - Added explicit parameters for `AIOHttpConnection` and `AsyncTransport` ([#276](https://github.com/opensearch-project/opensearch-py/pull/276))
 - Added support for a custom signing service name for AWS SigV4 ([#268](https://github.com/opensearch-project/opensearch-py/pull/268))
+- Added new guide for using index templates with the client ([#351](https://github.com/opensearch-project/opensearch-py/issues/351))
```

Comment:
@Djcarrillo6 Please write your **PR details** instead of issue details in the **unreleased - added section**

Reply:
@saimedhi this has been fixed and pushed.